schedule/yast/qam-yast_self_update.yaml

```diff
@@ -11,9 +11,8 @@ schedule:
   - installation/licensing/accept_license
   - installation/registration/register_via_scc
   - installation/module_registration/register_extensions_and_modules
-  - installation/add_update_test_repo
+  - installation/add_maintenance_repos
   - installation/addon_products_sle
```

Comment:
we don't need this test module anymore, right? or maybe you left it because following test modules are not libyui-rest-api and something failed? if so, it is ok, in other ticket we could just move all to libyui.